CHANGELOG.rst

```diff
@@ -8,6 +8,12 @@
   individuals, populations, or sites, aiming to change the tree sequence tables as
   little as possible.
 
+**Bugfixes**
+
+- The returned posteriors when ``return_posteriors=True`` now return actual
+  probabilities (scaled so that they sum to one) rather than normalised
```

Comment:
The use of "normalised probabilities" is a bit confusing here, because normalizing a probability distribution typically means scaling such that it sums/integrates to 1; not the sort of under/overflow-protection that is happening here